.travis.yml

```diff
@@ -7,6 +7,10 @@ matrix:
       smalltalk: Squeak-trunk
 
     - os: linux
+      language: smalltalk
+      smalltalk: Squeak-5.0
+
+    - os: linux
```

Comment:
Was this previously not possible because of the custom changes to the system package? If not, it seems somewhat out of place in this PR.